src/vmm/src/devices/virtio/iovec.rs

```diff
@@ -21,6 +21,8 @@ pub enum IoVecError {
     ReadOnlyDescriptor,
     /// Tried to create an `IoVec` or `IoVecMut` from a descriptor chain that was too large
     OverflowedDescriptor,
+    /// Tried to push to full IovDeque.
+    IovDequeOverflow,
```

Comment:
Could you please make all the `iovec.rs` changes in the commit that makes `IoVecBufferMut` to hold multiple `DescriptorChain`s?

Reply:
done